reana-ui/src/pages/workflowList/components/WorkflowDeleteModal.js

```diff
@@ -0,0 +1,75 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import { Button, Modal, Checkbox } from "semantic-ui-react";
+
+import { deleteWorkflow } from "../../../actions";
+
+export default function WorkflowDeleteModal({
+  open,
+  workflow,
+  setOpenDeleteModal,
+  setSelectedWorkflow,
+}) {
+  const dispatch = useDispatch();
+  const [deleteWorkspace, setDeleteWorkspace] = useState(true);
+
+  const onCloseModal = () => {
+    setOpenDeleteModal(false);
+    setSelectedWorkflow({});
+    setDeleteWorkspace(true);
+  };
+
+  const { id, name, run, size } = workflow;
+  return (
+    <Modal open={open} onClose={onCloseModal} closeIcon>
+      <Modal.Header>Delete workflow</Modal.Header>
+      <Modal.Content>
+        {
```

Comment:
Correct me if I'm wrong, but I think curly brackets are not needed here

Reply:
Indeed, leftover from an older version when I was only using prop shorthands.